front/components/vaults/CreateOrEditVaultModal.tsx

```diff
@@ -102,9 +102,21 @@ export function CreateOrEditVaultModal({
         id: "name",
         accessorKey: "name",
         cell: (info: Info) => (
-          <DataTable.CellContent avatarUrl={info.row.original.icon}>
-            {info.row.original.name}
-          </DataTable.CellContent>
+          <>
```

Comment:
Note: did this manually as it would be weird not to show the email while allowing to search on the email..